utils/README

```diff
@@ -77,3 +77,11 @@ sign_packages.sh
 ------------------------------------------------------------------------------
     The shell script used to automate package signing for releases.
 
+backup_sqlite.sh
```

Comment:
This entries look to be in alphabetical order (mostly). Can you put this entry with the other xxx_backup ones?